### PR TITLE
fix markdown bug: mark code element and ignore pre element.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @superyyrrzz @ansyral @qinezh @hellosnow @vicancy

### CODE REVIEW

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static class Inline
             public static readonly Regex Escape = new Regex(@"^\\([\\`*{}\[\]()#+\-.!_>])", RegexOptionCompiled, RegexTimeOut);
             public static readonly Regex Comment = new Regex(@"^<!--(?:[^-]|-(?!->))*-->", RegexOptionCompiled, RegexTimeOut);
             public static readonly Regex AutoLink = new Regex(@"^<([^ >]+(@|:\/)[^ >]+)>", RegexOptionCompiled, RegexTimeOut);
-            public static readonly Regex CodeElement = new Regex(@"^\<code\>[\s\S]*?\</code\>", RegexOptionCompiled | RegexOptions.IgnoreCase, RegexTimeOut);
+            public static readonly Regex PreElement = new Regex(@"^\<pre\>[\s\S]*?\</pre\>", RegexOptionCompiled | RegexOptions.IgnoreCase, RegexTimeOut);
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs]: apply inline rules.

Comment (vicancy) [inline src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs]: <code> *p = *q </code> get bold?

---
In reply to: [90398046](https://github.com/dotnet/docfx/pull/937#discussion_r90398046) [](ancestors = 90398046)

Comment (vicancy) [inline src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs]: <code>*p=*q</code> get bold?

Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs]: In GitHub, it gets bold in New File Preview, but doesn't in comment Preview here... It also gets bold in commonmark: http://spec.commonmark.org/dingus/?text=%3Ccode%3E*p%3D*q%3C%2Fcode%3E

**Comment by qinezh** [comment]
:shipit:

**Pull request merged**